sdk/spring/azure-spring-cloud-autoconfigure/pom.xml

```diff
@@ -210,10 +210,10 @@
             <bannedDependencies>
               <includes>
                 <include>com.fasterxml.jackson.core:jackson-core:[2.11.2]</include> <!-- {x-include-update;com.fasterxml.jackson.core:jackson-core;external_dependency} -->
-                <include>com.microsoft.azure:spring-cloud-azure-context:[1.3.0-beta.1]</include> <!-- {x-include-update;com.microsoft.azure:spring-cloud-azure-context;current} -->
-                <include>com.microsoft.azure:spring-cloud-azure-telemetry:[1.3.0-beta.1]</include> <!-- {x-include-update;com.microsoft.azure:spring-cloud-azure-telemetry;current} -->
-                <include>com.microsoft.azure:spring-integration-eventhubs:[1.3.0-beta.1]</include> <!-- {x-include-update;com.microsoft.azure:spring-integration-eventhubs;current} -->
-                <include>com.microsoft.azure:spring-integration-servicebus:[1.3.0-beta.1]</include> <!-- {x-include-update;com.microsoft.azure:spring-integration-servicebus;current} -->
+                <include>com.azure:azure-spring-cloud-starter-context:[1.3.0-beta.1]</include> <!-- {x-include-update;com.azure:azure-spring-cloud-starter-context;current} -->
+                <include>com.azure:azure-spring-cloud-starter-telemetry:[1.3.0-beta.1]</include> <!-- {x-include-update;com.azure:azure-spring-cloud-starter-telemetry;current} -->
+                <include>com.azure:azure-spring-integration-eventhubs:[1.3.0-beta.1]</include> <!-- {x-include-update;com.azure:azure-spring-integration-eventhubs;current} -->
+                <include>com.azure:azure-spring-integration-servicebus:[1.3.0-beta.1]</include> <!-- {x-include-update;com.azure:azure-spring-integration-servicebus;current} -->
```

Comment:
we can remove all the dependencies which the group id starts with `com.azure:xxx` in `bannedDependencies` part.